.changeset/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": ["@changesets/changelog-github", { "repo": "latticexyz/mud" }],
```

Comment:
I honestly have no idea what the difference is between this and the default changelog but wagmi used it and they are goated so copy pasta ftw

Reply:
I think maybe the default publishes to packages/my-package/CHANGELOG instead of github?

Reply:
the package doesn't have a readme but the google preview makes it sounds like this makes the changelog include links to github commits, PRs and users, which sounds useful
![CleanShot 2023-06-20 at 10 46 06](https://github.com/latticexyz/mud/assets/89248902/2c4638b3-dc85-4b5b-9e8d-d0d2165299cb)

Reply:
Oh nice! I think I was confused because I thought the default did that but looking now it doesn't.